packages/js/data/src/reviews/resolvers.ts

```diff
@@ -9,22 +9,29 @@ import { addQueryArgs } from '@wordpress/url';
 import { NAMESPACE } from '../constants';
 import { setError, updateReviews } from './actions';
 import { fetchWithHeaders } from '../controls';
+import { ReviewObject, ReviewsQueryParams } from './types';
 
-export function* getReviews( query ) {
+export function* getReviews( query: ReviewsQueryParams ) {
 	try {
 		const url = addQueryArgs( `${ NAMESPACE }/products/reviews`, query );
-		const response = yield fetchWithHeaders( {
+		const response: {
+			headers: Map< string, string >;
+			data: Array< ReviewObject >;
+		} = yield fetchWithHeaders( {
 			path: url,
 			method: 'GET',
 		} );
 
-		const totalCount = parseInt( response.headers.get( 'x-wp-total' ), 10 );
+		const totalCount = parseInt(
+			response.headers.get( 'x-wp-total' ) ?? '0',
```

Comment:
Do you think if we should throw an error when the value is NAN?

Reply:
Not too sure, that changes the behavior but so does returning 0 by default... Maybe its better to throw an error, will do that